.gitignore

```diff
@@ -0,0 +1,10 @@
+# Logs
+logs
+*.log
+
+# Dependency directories
+node_modules/
+bower_components/
+
+# Mac specific
+.DS_Store
```

Comment:
Have you thought about what you want to do for compiled files? Some repos ignore compiled files, build then on the CI server, like travis CI, and then somehow get them to the actual server. This is more setup but better separation. It's easier to just include the compiled files in the repo but be aware that you may get more merge conflicts because of them.